README.md

```diff
@@ -28,10 +28,16 @@ You can find all the documentation for LunarVim at [lunarvim.org](https://www.lu
 
 Make sure you have the release version of Neovim (0.6).
 
+Linux:
 ```bash
 bash <(curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/master/utils/installer/install.sh)
 ```
 
+Windows (Powershell):
+```powershell
+iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/LunarVim/LunarVim/master/utils/installer/install.ps1'))
```

Comment:
```suggestion
iwr https://raw.githubusercontent.com/LunarVim/LunarVim/master/utils/installer/install.ps1 | iex
```
Does this work? It's a bit shorter, and we can also expand both `iex` and `iwr`

Reply:
I had to add the `-UseBasicParsing` flag for `iwr` in Windows Sandbox, but I think PowerShell generally adds the flag by default. Other than that, it works!

Also, it would probably be a good idea to expand `iex` and `iwr` in case the aliases aren't set.